src/syntax/ppx_sedlex.ml

```diff
@@ -282,8 +289,12 @@ let regexp_of_pattern env =
         | Some r -> r
         | None ->
           err p.ppat_loc
-            "the Compl operator can only applied to a single-character regexp"
+            "the Compl operator can only applied to a single-character length regexp"
```

Comment:
The same expression is used multiple times in the doc, can you change it there as well ?